azkaban-common/src/main/java/azkaban/project/AzkabanProjectLoader.java

```diff
@@ -95,13 +97,12 @@
 
       reports = validateProject(project, archive, file, prop);
 
-      // Todo jamiesjc: in Flow 2.0, we need to create new flowLoader class and
-      // call new method to load the project flows.
-      // Need to guicify it later so that we can mock flowLoader in the tests.
-      // Load the project flows.
-      final DirectoryFlowLoader directoryFlowLoader = new DirectoryFlowLoader(prop);
-      reports.put(DIRECTORY_FLOW_REPORT_KEY,
-          directoryFlowLoader.loadProject(project, file));
+      // Todo jamiesjc: need to check if manifest file exists in project zip,
```

Comment:
Can this logic be encapsulated in loader class?

Reply:
Good idea, I'll move this logic into the loader class.